src/clarity/iconography/demo/icon-selection.demo.html

```diff
@@ -1,20 +1,148 @@
-<!--
-  ~ Copyright (c) 2016 VMware, Inc. All Rights Reserved.
-  ~ This software is released under MIT license.
-  ~ The full license information can be found in LICENSE in the root directory of this project.
-  -->
-
-<span class="clr-example">
-    <clr-icon shape="info"></clr-icon>
-    <clr-icon shape="success"></clr-icon>
-    <clr-icon shape="warning"></clr-icon>
-    <clr-icon shape="danger"></clr-icon>
-</span>
-
-<pre>
-<code clr-code-highlight="language-html">
-&lt;clr-icon shape=&quot;info&quot;&gt;&lt;/clr-icon&gt;
-&lt;clr-icon shape=&quot;success&quot;&gt;&lt;/clr-icon&gt;
-&lt;clr-icon shape=&quot;warning&quot;&gt;&lt;/clr-icon&gt;
-&lt;clr-icon shape=&quot;danger&quot;&gt;&lt;/clr-icon&gt;
-</code></pre>
+<div class="clr-icon-selection">
+    <div class="row">
+        <div class="clr-icon-item col-xs-12 col-sm-6 col-md-4 col-lg-3">
+            <clr-icon shape="home"></clr-icon><span>home</span>
+        </div>
+        <div class="clr-icon-item col-xs-12 col-sm-6 col-md-4 col-lg-3">
+            <clr-icon shape="menu"></clr-icon><span>menu</span>
```

Comment:
If we're going with describing the icon and not the usage, "menu" is probably not the right name.